src/test/suite/CamelKSchemaManager.test.ts

```diff
@@ -50,11 +50,11 @@ suite('Test Camel K Schema Manager', function () {
 		expect(schema).to.be.undefined;
 	});
 
-	test('Change setting for the automatic schema binding', async () => {
+	test.only('Change setting for the automatic schema binding', async () => {
```

Comment:
`.only` ?

Reply:
oh that I feel stupid, second time in a week